### ISSUE TITLE
Switching tabs in active panel only

### ISSUE THREAD

**Issue by rob-milicka** [mbnuqw/sidebery#942]
## Description
Hi, would it be possible to implement feature where If I am switching tabs by ctrl+tab shortcut would not switch to other panels? I would like to keep switching tabs just in active panel and I do not see the option how to do that. Thanks!

**Comment by mbnuqw** [comment]
No, it's not possible, because Firefox don't allow handling the internal shortcuts (like ctrl+Tab). But I can add Sidebery's keybinding for that action (switching to next/previous tab in panel) and you will be able to set different shortcut for it (it also not possible to reassign shortcuts used by Firefox itself).

**Comment by rob-milicka** [comment]
Switching to next/previous tab in panel / That would be great. Thanks for quick answer and also for your work on Sidebery! Happy to use it ;-)

**Comment by magnus-ISU** [comment]
Any chance we can get this? I think this would make panels a lot more useful - as it is now, I accidentally go back/forth between them often

**Comment by pwall2222** [comment]
Would this implementation work? (or something close to this)

`src/services/keybindings.actions.ts`
```ts
function onKeyNextTabInPanel() {
  const actTab = Tabs.byId[Tabs.activeId]
  if (!actTab?.panelId) return
  const panel = Sidebar.panelsById[actTab?.panelId]
  if (!Utils.isTabsPanel(panel)) return
  

  let index = panel.tabs.indexOf(actTab) + 1
  if (index == panel.tabs.length) index = 0
  
  const nextTabId = panel.tabs[index].id

  browser.tabs.update(nextTabId, { active: true })
}
```
